database/src/main/java/com/firebase/ui/database/FirebaseListAdapter.java

```diff
@@ -73,6 +73,11 @@ public FirebaseListAdapter(Activity activity, Class<T> modelClass, int modelLayo
             public void onChanged(EventType type, int index, int oldIndex) {
                 notifyDataSetChanged();
             }
+
+            @Override
+            public void onCancelled(DatabaseError databaseError) {
+                FirebaseRecyclerAdapter.this.onCancelled(databaseError);
```

Comment:
I think you mean `FirebaseListAdapter.this` here, (rather than `FirebaseRecyclerAdapter.this`)